Files/Views/YourHome.xaml.cs

```diff
@@ -165,5 +165,16 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
             };
             AppInstance.NavigationToolbar.PathComponents.Add(item);
         }
+
+        #region IDisposable
+
+        // TODO: This Dispose() is never called, please implement the functionality to call this function.
+        //       This IDisposable.Dispose() needs to be called to unhook events in BundlesWidget to avoid memory leaks.
```

Comment:
This one is important. Currently we have a memory leak here because this `IDisposable.Dispose()` is not called.

Reply:
The solution is already in place but `YourHome.Dispose()` needs to be called which exceedes the scope of this PR :)